tool/xtt.c

```diff
@@ -51,6 +52,19 @@ int main(int argc, char **argv)
             out = xtt_generate_server_certificate(params.rootcert, params.rootpriv, params.serverpriv,
                                                 params.serverpub, params.server_id, params.time, params.servercert);
             break;
+        case action_infocert: {
+            enum infocert_type type;
+            if (params.servercert != NULL) {
+                type = infocert_type_server;
+                out = info_cert(params.servercert, type);
+            } else if (params.rootcert != NULL){
```

Comment:
Inconsistent spacing here.  Line 55 has a space before the `{`; this line doesn't.  

I personally prefer spaces before braces, but you be consistent with whatever the standard is for `xtt`.